unleash/src/main/java/com/silvercar/unleash/repository/HttpToggleFetcher.kt

```diff
@@ -0,0 +1,80 @@
+package com.silvercar.unleash.repository
+
+import com.silvercar.unleash.UnleashException
+import com.silvercar.unleash.repository.FeatureToggleResponse.Status.CHANGED
+import com.silvercar.unleash.repository.FeatureToggleResponse.Status.NOT_CHANGED
+import com.silvercar.unleash.repository.FeatureToggleResponse.Status.UNAVAILABLE
+import com.silvercar.unleash.util.UnleashConfig
+import java.io.BufferedReader
+import java.io.IOException
+import java.io.InputStream
+import java.io.InputStreamReader
+import java.net.HttpURLConnection
+import java.net.URL
+import kotlin.text.Charsets.UTF_8
+
+class HttpToggleFetcher(private val unleashConfig: UnleashConfig) : ToggleFetcher {
+  private var eTag = ""
+  private val toggleUrl: URL = unleashConfig.unleashURLs.fetchTogglesURL
+  private val jsonToggleParser: JsonToggleParser = JsonToggleParser()
```

Comment:
@deandalm Same here, could we simply set this line as `private val jsonToggleParser = JsonToggleParser()`?